### ISSUE TITLE
Rails 4 support

### ISSUE THREAD

**Issue by frabrunelle** [dailycred/dailycred#5]
I can't get this gem to work in Rails 4. First, I changed "match" for "get" in config/routes.rb, because "match" is deprecated in Rails 4. But then I get this error when I try to create a new user:
### can't write unknown attribute `attributes'

``` Ruby
activerecord (4.0.0.beta1) lib/active_record/attribute_methods/write.rb:45:in `write_attribute'
activerecord (4.0.0.beta1) lib/active_record/attribute_methods/dirty.rb:70:in `write_attribute'
activerecord (4.0.0.beta1) lib/active_record/attribute_methods.rb:303:in `[]='
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/gems/2.0.0/bundler/gems/dailycred-7f74bdf5bb71/lib/dailycred/acts_as_dailycred.rb:55:in `block in update_from_dailycred'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/gems/2.0.0/bundler/gems/dailycred-7f74bdf5bb71/lib/dailycred/acts_as_dailycred.rb:54:in `each'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/gems/2.0.0/bundler/gems/dailycred-7f74bdf5bb71/lib/dailycred/acts_as_dailycred.rb:54:in `update_from_dailycred'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/gems/2.0.0/bundler/gems/dailycred-7f74bdf5bb71/lib/dailycred/acts_as_dailycred.rb:15:in `find_or_create_with_omniauth'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/gems/2.0.0/bundler/gems/dailycred-7f74bdf5bb71/app/controllers/sessions_controller.rb:8:in `create'
actionpack (4.0.0.beta1) lib/action_controller/metal/implicit_render.rb:4:in `send_action'
actionpack (4.0.0.beta1) lib/abstract_controller/base.rb:189:in `process_action'
actionpack (4.0.0.beta1) lib/action_controller/metal/rendering.rb:10:in `process_action'
actionpack (4.0.0.beta1) lib/abstract_controller/callbacks.rb:18:in `block in process_action'
activesupport (4.0.0.beta1) lib/active_support/callbacks.rb:423:in `_run__1046491453522648181__process_action__callbacks'
activesupport (4.0.0.beta1) lib/active_support/callbacks.rb:78:in `run_callbacks'
actionpack (4.0.0.beta1) lib/abstract_controller/callbacks.rb:17:in `process_action'
actionpack (4.0.0.beta1) lib/action_controller/metal/rescue.rb:29:in `process_action'
actionpack (4.0.0.beta1) lib/action_controller/metal/instrumentation.rb:31:in `block in process_action'
activesupport (4.0.0.beta1) lib/active_support/notifications.rb:158:in `block in instrument'
activesupport (4.0.0.beta1) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
activesupport (4.0.0.beta1) lib/active_support/notifications.rb:158:in `instrument'
actionpack (4.0.0.beta1) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
actionpack (4.0.0.beta1) lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
activerecord (4.0.0.beta1) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
actionpack (4.0.0.beta1) lib/abstract_controller/base.rb:136:in `process'
actionpack (4.0.0.beta1) lib/abstract_controller/rendering.rb:44:in `process'
actionpack (4.0.0.beta1) lib/action_controller/metal.rb:196:in `dispatch'
actionpack (4.0.0.beta1) lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
actionpack (4.0.0.beta1) lib/action_controller/metal.rb:232:in `block in action'
actionpack (4.0.0.beta1) lib/action_dispatch/routing/route_set.rb:78:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/routing/route_set.rb:78:in `dispatch'
actionpack (4.0.0.beta1) lib/action_dispatch/routing/route_set.rb:46:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/journey/router.rb:69:in `block in call'
actionpack (4.0.0.beta1) lib/action_dispatch/journey/router.rb:57:in `each'
actionpack (4.0.0.beta1) lib/action_dispatch/journey/router.rb:57:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/routing/route_set.rb:651:in `call'
railties (4.0.0.beta1) lib/rails/engine.rb:510:in `call'
railties (4.0.0.beta1) lib/rails/railtie/configurable.rb:30:in `method_missing'
actionpack (4.0.0.beta1) lib/action_dispatch/journey/router.rb:69:in `block in call'
actionpack (4.0.0.beta1) lib/action_dispatch/journey/router.rb:57:in `each'
actionpack (4.0.0.beta1) lib/action_dispatch/journey/router.rb:57:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/routing/route_set.rb:651:in `call'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/gems/2.0.0/bundler/gems/dailycred-7f74bdf5bb71/lib/dailycred/middleware.rb:17:in `call'
omniauth (1.1.3) lib/omniauth/strategy.rb:394:in `call_app!'
omniauth (1.1.3) lib/omniauth/strategy.rb:356:in `callback_phase'
omniauth-oauth2 (1.1.1) lib/omniauth/strategies/oauth2.rb:77:in `callback_phase'
omniauth (1.1.3) lib/omniauth/strategy.rb:219:in `callback_call'
omniauth (1.1.3) lib/omniauth/strategy.rb:175:in `call!'
omniauth (1.1.3) lib/omniauth/strategy.rb:157:in `call'
omniauth (1.1.3) lib/omniauth/builder.rb:48:in `call'
rack (1.5.2) lib/rack/etag.rb:23:in `call'
rack (1.5.2) lib/rack/conditionalget.rb:25:in `call'
rack (1.5.2) lib/rack/head.rb:11:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/flash.rb:241:in `call'
rack (1.5.2) lib/rack/session/abstract/id.rb:225:in `context'
rack (1.5.2) lib/rack/session/abstract/id.rb:220:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/cookies.rb:452:in `call'
activerecord (4.0.0.beta1) lib/active_record/query_cache.rb:36:in `call'
activerecord (4.0.0.beta1) lib/active_record/connection_adapters/abstract/connection_pool.rb:632:in `call'
activerecord (4.0.0.beta1) lib/active_record/migration.rb:348:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
activesupport (4.0.0.beta1) lib/active_support/callbacks.rb:373:in `_run__890670898852174863__call__callbacks'
activesupport (4.0.0.beta1) lib/active_support/callbacks.rb:78:in `run_callbacks'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/reloader.rb:64:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
railties (4.0.0.beta1) lib/rails/rack/logger.rb:38:in `call_app'
railties (4.0.0.beta1) lib/rails/rack/logger.rb:21:in `block in call'
activesupport (4.0.0.beta1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
activesupport (4.0.0.beta1) lib/active_support/tagged_logging.rb:25:in `tagged'
activesupport (4.0.0.beta1) lib/active_support/tagged_logging.rb:67:in `tagged'
railties (4.0.0.beta1) lib/rails/rack/logger.rb:21:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
rack (1.5.2) lib/rack/runtime.rb:17:in `call'
activesupport (4.0.0.beta1) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
rack (1.5.2) lib/rack/lock.rb:17:in `call'
actionpack (4.0.0.beta1) lib/action_dispatch/middleware/static.rb:64:in `call'
railties (4.0.0.beta1) lib/rails/engine.rb:510:in `call'
railties (4.0.0.beta1) lib/rails/application.rb:96:in `call'
rack (1.5.2) lib/rack/lock.rb:17:in `call'
rack (1.5.2) lib/rack/content_length.rb:14:in `call'
rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
/Users/frabrunelle/.rbenv/versions/2.0.0-p0/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'
```

I'm a noob so I don't really understand how to fix this error. Anyways, just let me know if you're able to get this gem working in Rails 4. Thank you.

**Comment by hstove** [comment]
fixed with 0e735053335908032ceff7b043d349c9f2170867

:heart: 

specify in your gemfile: `gem 'dailycred', '~> 0.1.7'`

**Comment by frabrunelle** [comment]
Awesome! I can confirm that it works perfectly well now :+1:
